lib/parallel_tests/rspec/runner.rb

```diff
@@ -48,6 +49,20 @@ def command_with_seed(cmd, seed)
           "#{clean} --seed #{seed}"
         end
 
+        # Summarize results from threads and colorize results based on failure and pending counts.
+        #
+        def summarize_results(results)
+          sums = sum_up_results(results)
+          color = if sums['failure'].positive?
+                    :failure
+                  elsif sums['pending'].positive?
+                    :pending
+                  else
+                    :success
+                  end
+          results_string = sums.sort.map { |word, number| "#{number} #{word}#{'s' if number != 1}" }.join(', ')
```

Comment:
```suggestion
          text = super
          return text unless $stdout.tty?
          color =
            if sums['failure'].positive?
               31 # red
            elsif sums['pending'].positive?
              33 # yellow
            else
               32 # green
            end
          "\e[#{color}m#{text}\e[0m"
```

Reply:
... and even then how about moving this into lib/parallel_tests/test/runner.rb since this is pretty universal ?

Reply:
... and there is a good chance some of these keys will be missing, so doing something like `sums.key?('pending')` might be safer